src/Orchard.Web/Modules/Orchard.Projections/Migrations.cs

```diff
@@ -262,16 +266,26 @@ public class Migrations : DataMigrationImpl {
 
             ContentDefinitionManager.AlterTypeDefinition("ProjectionPage", cfg => cfg.Listable());
 
-            return 3;
+            return 4;
         }
 
         public int UpdateFrom2() {
-            SchemaBuilder.AlterTable("ProjectionPartRecord",
-                            table => table
-                                .AlterColumn("PagerSuffix", c => c.WithType(DbType.String).WithLength(255))
-                            );
+            SchemaBuilder.AlterTable("ProjectionPartRecord", table => table
+                .AlterColumn("PagerSuffix", c => c.WithType(DbType.String).WithLength(255))
+            );
 
             return 3;
         }
+
+        public int UpdateFrom3() {
+            SchemaBuilder.AlterTable("LayoutRecord", table => table
+                .AddColumn<string>("Guid", c => c.WithLength(64))

```

Comment:
Oh good! I'll change then. I originally wanted to specify 32 characters, but that isn't enough because of the hyphens. I either had to skip the hyphens or count the actual number of characters, so I decided to be lazy and just go with 64. I love nitpicking!